src/components/UsingConfigCatClient.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="using-configcat-client">
+    <div class="flags-not-found" v-if="featureFlags === undefined || featureFlags.length === 0">
```

Comment:
nit: The if condition can be simplified as `0` is a falsy value in JS.

```suggestion
    <div class="flags-not-found" v-if="!featureFlags?.length">
```